src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -224,7 +242,47 @@ public void BindingWorkflow_InstallPackages_FailureOnOneProject_Continues()
             // Verify
             packageInstaller.AssertNoInstalledPackages(project1);
             packageInstaller.AssertInstalledPackages(project2, packages);
-            outputWindowPane.AssertOutputStrings(string.Format(CultureInfo.CurrentCulture, Strings.FailedDuringNuGetPackageInstall, nugetPackage.Id, project1Name, failureMessage));
+            this.outputWindowPane.AssertOutputStrings(4);
+            this.outputWindowPane.AssertOutputStrings(
+                string.Format(Strings.SubTextPaddingFormat, string.Format(Strings.EnsuringNugetPackagesProgressMessage, nugetPackage.Id, project1Name)),
+                string.Format(Strings.SubTextPaddingFormat, string.Format(Strings.FailedDuringNuGetPackageInstall, nugetPackage.Id, project1Name, failureMessage)),
+                string.Format(Strings.SubTextPaddingFormat, string.Format(Strings.EnsuringNugetPackagesProgressMessage, nugetPackage.Id, project2Name)),
+                string.Format(Strings.SubTextPaddingFormat, string.Format(Strings.SuccessfullyInstalledNugetPackageForProject, nugetPackage.Id, project2Name))
+                );
+        }
+
+        [TestMethod]
+        public void BindingWorkflow_InstallPackages_WhenProjectLanguageDoesNotExist_PrintMessageAndContinue()
+        {
+            // Setup
+            const string project1Name = "project1";
+            const string project2Name = "project2";
+
+            var testSubject = this.CreateTestSubject();
+            var progressEvents = new ConfigurableProgressStepExecutionEvents();
+
+            ProjectMock project1 = new ProjectMock(project1Name);
```

Comment:
What happens when `ProjectKind` is not set? Is it going to be a non-supported project? Maybe add an explanation comment here.